server/db/models/user.js

```diff
@@ -26,6 +38,45 @@ const User = db.define('user', {
   },
   googleId: {
     type: Sequelize.STRING
+  },
+  street: {type: Sequelize.STRING},
+  city: {type: Sequelize.STRING},
+  state: {type: Sequelize.CHAR(2)},
+  postalCode: {type: Sequelize.FLOAT(5)},
```

Comment:
Should this be a float or an integer? Postal codes are whole numbers.

Reply:
fixed